### ISSUE TITLE
issue with font on blog

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hood.ie#296]
@danielberndt mentioned that the font on our blog could be improved for non-retina displays
- https://twitter.com/D40B/status/738370978951073792
- https://twitter.com/D40B/status/738371367901532160

**Comment by NickColley** [comment]
Could consider rendering only headings with the custom font.

**Comment by verpixelt** [comment]
I'd love to see screenshots of a non retina screen if possible. Changing the font on the blog/the entire website is a rather big operation and imho not worth it.

**Comment by danielberndt** [comment]
Here's a screenshot from a recent chrome window on my external monitor using OS X 10.11.5.

![blog-digital-screenshot](https://cloud.githubusercontent.com/assets/856386/15855340/378ddaba-2caf-11e6-946a-ddbce07b9ff0.png)

As examples have a look at the two `ll` from the initial `Well`, and at the horizontal line of any `e`. Not sure whether the font lacks proper hinting, or it is a general issue with any font of that width.

This issue is certainly not the end of the world, but once you start noticing the lack crispness, it's kinda hard to solely focus on the content. (Which is great, btw! :) )

**Comment by danielberndt** [comment]
Just noticed: seeing the screenshot on a retina screen certainly gives the wrong impression since the image gets blurred. So I recommend downloading it, and zooming into it, like this: 

<img width="276" alt="screen shot 2016-06-07 at 13 07 40" src="https://cloud.githubusercontent.com/assets/856386/15855627/d45091ca-2cb0-11e6-9a2c-2770a01f6063.png">

**Comment by chrisdemars** [comment]
Is anyone looking in to this issue?

**Comment by gr2m** [comment]
nope, take it away if you like :) but note that we have #312 coming up which will be a redesign. But doing some research about this issue might be helpful either way

**Comment by chrisdemars** [comment]
Awesome
